Marlin/Conditionals.h

```diff
@@ -45,11 +45,27 @@
     #define DOGLCD  // Support for I2C LCD 128x64 (Controller SSD1306 graphic Display Family)
   #endif
 
-
   #if ENABLED(PANEL_ONE)
     #define ULTIMAKERCONTROLLER
   #endif
 
+  #if ENABLED(BQ_LCD_SMART_CONTROLLER)
+    #define REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER
+    #define SDSUPPORT
```

Comment:
`SDSUPPORT` is now an option that users must enable explicitly. It was changed so that it can simply be left disabled to turn `SDSUPPORT` off. So you should leave it out here, but enable it in the `Hephestos_2/Configuration.h ` file.